### DISCUSSION TITLE
Request

### DISCUSSION THREAD

**Discussion by stevebaines** [paulmillr/encrypted-dns#79] [General]
Hi Paul,  
May I request a DoH and DoT profile for NextDNS.  I can input the IP addresses but I am stumped by where the PayloadUUID comes from.

**Comment by zecanard** [comment]
Open a Terminal session and simply enter `uuidgen`. Or use an online generator like https://www.uuidgenerator.net/, which can create them in bulk (I think these profiles need 2 UUIDs each).

**Reply by stevebaines** [reply]
Super helpful. Thank you 🙏🏼
